Source/WebCore/platform/graphics/filters/FilterOperations.cpp

```diff
@@ -175,6 +175,33 @@ bool FilterOperations::hasFilterThatShouldBeRestrictedBySecurityOrigin() const
     return false;
 }
 
+bool FilterOperations::canInterpolate(const FilterOperations& to, CompositeOperation compositeOperation) const
```

Comment:
Does this apply to all implementations, or does it describe specific limitations of Core Animation based ones? If so, it should live in platform code.

Reply:
No, this is the rule as specified by CSS Filters, the link is just a little below.